front/src/common/Map/Layers/GeoJSONs.tsx

```diff
@@ -334,10 +335,10 @@ const SOURCES_DEFINITION: {
   { entityType: 'errors', getLayers: getErrorsLayers },
 ];
 
-export const SourcesDefinitionsIndex = mapValues(
-  keyBy(SOURCES_DEFINITION, 'entityType'),
-  (def) => def.getLayers
-) as Record<LayerType, (context: LayerContext, prefix: string) => AnyLayer[]>;
+export const SourcesDefinitionsIndex = SOURCES_DEFINITION.reduce(
+  (acc, curr) => ({ ...acc, [curr.entityType]: curr.getLayers }),
+  {} as Record<LayerType, (context: LayerContext, prefix: string) => AnyLayer[]>
+);
```

Comment:
Does it change something or it's just not to depend on lodash ?

Reply:
During my investigation, I  changed it to be able to add an `order` props, but it was not the good way to solve the issue. 
And then I left it like that... Moreover, IMHO it's easier to read the reduce like that than to have two lodash functions (but it's a personal view)

Reply:
totally agreed :)

Reply:
Agree too !